api/handlers.go

```diff
@@ -1104,6 +1104,7 @@ func (si *ServerImplementation) fetchApplicationBoxes(ctx context.Context, param
 			if appid == 0 {
 				appid = generated.ApplicationId(result.App)
 			}
+			result.Box.Round = uint64(round)
```

Comment:
```suggestion
			result.Box.Round = round
```
`round` is already a `uint64`

Reply:
[b17fbd7](https://github.com/algorand/indexer/pull/1519/commits/b17fbd75009c974d09f73404c6f3038e9cf1ed85)